src/bin/electrscash.rs

```diff
@@ -62,12 +62,28 @@ fn run_server(config: &Config) -> Result<()> {
     let query = Query::new(app.clone(), &metrics, tx_cache, config.txid_limit);
 
     let mut server = None; // Electrum RPC server
+
+    // If we see this more new blocks than this, don't look for scripthash
+    // changes. Just have clients reconnect.
+    //
+    // This many header changes means we're either not fully synced, or there
+    // has been abnormally large reorg of the blockchain.
+    const MAX_SCRIPTHASH_BLOCKS: usize = 20;
```

Comment:
This hack was removed and improved in https://github.com/BitcoinUnlimited/ElectrsCash/pull/53